components/mpas-ocean/src/analysis_members/mpas_ocn_moc_streamfunction.F

```diff
@@ -620,10 +626,13 @@ subroutine ocn_compute_moc_streamfunction(domain, timeLevel, err)!{{{
               do k = 1,maxLevelCell(iCell)
                  do i = 1, regionsInAddGroup
                     currentRegion = regionsInGroup(i, regionGroupNumber)
-                    sumVertBinVelocityRegion(iBin, k, i) = sumVertBinVelocityRegion(iBin, k, i) + (vertGMBolusVelocityTop(k, iCell) * &
+                    sumVertBinVelocityRegion(iBin, k, i) = sumVertBinVelocityRegion(iBin, k, i) + &
+                          ((vertGMBolusVelocityTop(k, iCell) +vertMLEBolusVelocityTop(k,iCell)) * &
                           areaCell(iCell) * regionCellMasks(currentRegion, iCell))
                  end do
-                 sumVertBinVelocity(iBin, k) = sumVertBinVelocity(iBin, k) + (vertGMBolusVelocityTop(k, iCell) * areaCell(iCell))
+                 sumVertBinVelocity(iBin, k) = sumVertBinVelocity(iBin, k) + &
+                          ((vertGMBolusVelocityTop(k, iCell) + vertMLEBolusVelocityTop(k,iCell)) &
+                          * areaCell(iCell))
```

Comment:
Same question about whether these 3D fields are worth the memory they take up for configurations that don't use them.